src/it/resources/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/InputFastMatcher.java

```diff
@@ -1,4 +1,4 @@
-package com.google.checkstyle.test.chapter4formatting.rule42blockindentaion;
+package com.google.checkstyle.test.chapter4formatting.rule42blockindentation;
```

Comment:
@romani I see no block indentation for rule 4.2 on google's style. http://checkstyle.sourceforge.net/google_style.html#a4.2
Current 4.2 is Indentation and test link point to 4.8.4.1.
There is no 4.2 tests under https://github.com/checkstyle/checkstyle/tree/master/src/it/java/com/google/checkstyle/test/chapter4formatting/ .

Reply:
Good catch, but this is separate issue, please register it.